engine_config.json

```
@@ -1,10 +1,11 @@
 {
     "WindowSize": [
-        1920,
-        1080
+        1920.0,
```

Why is this not a integer?
```suggestion
        1920.0,
        1080.0
```